fastlane/Fastfile

```diff
@@ -12,22 +12,22 @@
 
 default_platform(:ios)
 
-files_with_version_number = [
-  './RevenueCat.podspec',
-  './Sources/Misc/SystemInfo.swift',
-  './.jazzy.yaml',
-  './.version',
-  './scripts/docs/index.html',
-  './scripts/docs/v4_api_migration_guide.html',
-  './Sources/Misc/SystemInfo.swift'
-]
-files_with_version_number_without_prerelease_modifiers = [
-  './Sources/Info.plist',
-  './Tests/BackendIntegrationTestApp/Info.plist',
-  './Tests/BackendIntegrationTests/Info.plist',
-  './Tests/UnitTests/Info.plist',
-  './Tests/UnitTestsHostApp/Info.plist'
-]
+files_with_version_number = {
+  './RevenueCat.podspec' => ['s.version          = "{x}"'],
```

Comment:
Hmm feels strange to bake in the spaces here... But I guess it's ok.

Reply:
yes... I think we should be moving to real regex matching semver versions at some point so we can avoid this